src/sql/workbench/api/common/extHostDataProtocol.ts

```diff
@@ -27,7 +27,7 @@ export class ExtHostDataProtocol extends ExtHostDataProtocolShape {
 
 	constructor(
 		mainContext: IMainContext,
-		private uriTransformer: IURITransformer
+		private uriTransformer: IURITransformer | null
```

Comment:
Are URI Transformers actually going to have this duality of sometimes being null?